app/views/layouts/_navbar.html.erb

```diff
@@ -1,55 +1,93 @@
-<nav class="navbar navbar-inverse" data-turbolinks-permanent>
-  <div class="container-fluid">
-    <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
-      <a class="navbar-brand lobster" href="/">CoPosition</a>
-    </div>
-
-    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-2">
-      <ul class="nav navbar-nav">
-        <li><a href="<%= signed_in? ? developers_console_path : new_developer_session_path %>">Developers</a></li>
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">About <span class="caret"></span></a>
-          <ul class="dropdown-menu" role="menu">
-            <li><a href="#">Security</a></li>
-            <li><a href="#">Enrolled companies</a></li>
-            <li><a href="#">Why use CoPosition</a></li>
-            <li class="divider"></li>
-            <li><%= link_to("API", api_path) %></li>
-          </ul>
-        </li>
+<nav class="nav-wrapper red lighten-1" role="navigation" data-turbolinks-permanent>
```

Comment:
Would prefer if instead of overriding the classes on individual elements, if you used the SASS variables/overrides instead.

For example: 
```sass
$primary-color: color("red", "lighten-1");
```

Unless there's a specific reason for this?